blamo/lib/Data/stateDataExampleObj.json

```diff
@@ -0,0 +1,96 @@
+{
+  "StateData": {
+    "currentRoute": "currentRoute",
+    "randomNumber": 0,
+    "storage": {
+      "devicePath": "local/device/path",
+      "fileName": "currentFile",
+      "pathExtension": "CurrentPathExtension",
+      "nameIterator": 5
+    },
+    "list": ["Document0","Document1","Document2"],
+    "document": [
+      {
+        "document1":{
+          "logInfo": {
+            "projectName": "Project Name",
+            "startDate": "Date",
+            "endDate": "date",
+            "driller": "driller",
+            "projectGeologist": "Project Geologist",
+            "recorder": "Recorder",
+            "northing": "northing",
+            "easting": "Easting",
+            "highway": "Highway",
+            "county": "County",
+            "purpose": "Purpose",
+            "holeNo.": 1,
+            "E.A No.": 2,
+            "keyNo.": 4,
+            "startCard No.": 6,
+            "groundElevation(m)": 200,
+            "tubeHeight": 45,
+            "equipment": "Equipment",
+            "objectID": "objectID",
+            "testType": "testType",
+            "project": "project",
+            "number": "number",
+            "client": "client",
+            "lat": "lat",
+            "long": "long",
+            "location": "Location",
+            "elevationDatum": "elevationDatum",
+            "boreholeID": "BoreholeID",
+            "surfaceElevation": "surfaceElevation",
+            "contractor": "Contractor",
+            "method": "Method",
+            "loggedBy": "loggedBy",
+            "checkedBy": "checkedBy",
+            "fakedata": "FakeData"
+          },
+          "Tests": [
+            {
+              "Test1":{
+                "beginTestDepth": 5,
+                "endTestDepth": 6,
+                "soilType": "SoilType",
+                "description": "Description",
```

Comment:
Description here should be the same implementation as tags in Units.

Reply:
Not entirely sure what you mean. This is just going off of what you did at line 34 in pdf_classes.dart where it has a description String
![image](https://user-images.githubusercontent.com/43584732/75128327-6774e600-5678-11ea-9027-5bb9a7701a54.png)

Reply:
Tests and Units use the same style of selecting description tags from a set. In pdf_classes, Test.description and Unit.unitDescription store a value of dummy tags condensed into one string, currently initialized in pdf_dummy_data for the sake of having data to print. We need your method of implementing tags used in your unit.dart object to be in test.dart as well.